src/tests/ftest/util/performance_test_base.py

```diff
@@ -228,26 +202,50 @@ def set_processes_ppn(self, namespace):
 
         """
         self.processes = self.params.get("np", namespace, self.processes)
-        self.ppn = self.params.get("ppn", namespace, self.ppn)
-
-    def verify_system_status(self):
-        """Verify container/pool/system status (in that order)."""
-        if self.pool and self.container:
-            self.log.info(self.daos_cmd.container_query(self.pool.uuid, self.container.uuid))
-        if self.pool:
-            self.display_pool_space()
-        self.dmg_cmd.system_query()
-
-    def verify_oclass_server_count(self, oclass):
-        """Verify an object class is compatible with the number of servers.
+        self.ppn = self.perf_params.ppn = self.params.get("ppn", namespace, self.ppn)
+
+    def verify_system_status(self, pool=None, container=None):
+        """Verify system/pool/container status (in that order)."""
```

Comment:
You might want to add `pool` and `container` in the Args section in the comment.

Reply:
Gah. I forgot that :( Thanks!